test/Makefile

```diff
@@ -81,6 +81,7 @@ endif
 include ../src/osmodel.mak
 
 export OS
+BUILD=release
```

Comment:
> Why is this necessary? 

Because you need to find the DMD binary somehow and as Makefile variable at least the user can overwrite it...

> debug is a better choice for testing in general and tests for specific optimization arguments

Hehe `debug` is simply a different folder as of now:
https://github.com/dlang/dmd/pull/6898

> This variable affects only the test runner, right?

Yes.